src/components/FormInput.js

```diff
@@ -18,13 +16,15 @@ function FormInput({ name }) {
     <div>
       <button className="button-unfolding" onClick={handleToggleButton}>
         <FiPlus className="button-plus" />
-        <span className="button-label">{name}</span>
+        <label className="button-label">{name}</label>
       </button>
       <textarea
         id="input"
         className={`${classOnClick}`}
```

Comment:
Instead of :
```javascript
const classOnClick = isShown ? "input-hidden" : "input-shown";
```
a few lines up, you could have this here instead:
```javascript
className={isShown ? "input-hidden" : "input-shown"}
```
saves a variable :)

Reply:
Amazing! That was actually what the way I wanted it to look like! :)